six/modules/c++/six.sicd/include/six/sicd/Utilities.h

```diff
@@ -125,14 +125,19 @@ class Utilities
      *         the buffer pointer is null
      *
      */
+     static void getWidebandData(NITFReadControl& reader,
+                                const ComplexData& complexData,
+                                size_t startRow, size_t numRows,
```

Comment:
Can you change all these to pass in two `types::RowCol` instead: `const types::RowCol<size_t>& offset, const types::RowCol<size_t>& extent`.  Python layer is fine the way it is.

Also add Doxygen comments to the functions you're adding rather than grouping this function next to the old one